New Test: 2983_Enable/Disable Themes from about:addons.

ranjit-exalt opened the pull request:
Scripting completed for issue: #2983

Please review and merge.

Commit:
Applying this patch will add a new test case "enable_disable_themes_from_about_addons" (test_case_id="14991").

spronoza reviewed (changes requested):
Failing on Linux
![image](https://user-images.githubusercontent.com/46671906/72809143-bb4c7380-3c63-11ea-89fa-b1439084ec11.png)

Commit:
Changes log:
1. To resolve conflict merged latest changes of dev to issue_2983.
2. Improved dragged and drop functionality for "enable_disable_themes_from_about_addons"
3. Added one image (new_tab_icon_light) for each OS.

ranjit-exalt commented:
> Failing on Linux
> ![image](https://user-images.githubusercontent.com/46671906/72809143-bb4c7380-3c63-11ea-89fa-b1439084ec11.png)

Please review.
